### PR TITLE
Only add "Updating Bazel" link on master or > 0.28 docs.

### LINKED ISSUE #9056: "Updating Bazel" link on https://docs.bazel.build/versions/0.28.0/bazel-overview.html 404s
The leftnav bar is not versioned with the rest of the documentation, and if the link target in the navbar does not exist in a particular version,  the link 404s.

It has happened  with recently added https://docs.bazel.build/versions/master/updating-bazel.html (added in https://github.com/bazelbuild/bazel/commit/4285b4bf14eecbd0f5b7a0b09cad617109c6abab)

### PR DESCRIPTION
**Pull request by dslomov**
Fixes #9056.

### CODE REVIEW

**Comment by dslomov** [comment]
Principled fix would be versioning of _layouts/documentation.html, but it is more work and I have no quick idea how to start with that.

File: site/_layouts/documentation.html [hunk 1]
```diff
@@ -189,7 +199,8 @@ <h3>Rules</h3>
                 </ul>
               </li>
 
-              <li>
+              <li>{%   assign minor_version = version_components[0] %}
```
Comment (laszlocsomor) [inline site/_layouts/documentation.html]: could you explain why this is needed?

Comment (dslomov) [reply site/_layouts/documentation.html]: accident, removed

**Comment by jin** [comment]
The versioning problem of _layouts.documentation.html is tracked in https://github.com/bazelbuild/bazel/issues/6937. This seems like a good workaround for versioning the sidebar until we find a better solution.

**Pull request closed without merging**